### PR TITLE
change default install location to program files

### PR DESCRIPTION
**Pull request by pemba1s1**
**Issue** #77 #66

## Summary by Sourcery

Update the installation script to set the default install location to Program Files and refactor file path handling to improve clarity and maintainability.

Enhancements:
- Refactor file path retrieval to use a new utility function that determines the base path based on the application context.

Build:
- Change the default installation directory from AppData to Program Files in the installer script.

### CODE REVIEW

**Comment by ItsSimko** [comment]
@pemba1s1 This will need further revision as when the install is in program files the application doesn't have permission to write/read the settings.txt. So we will need to store the settings.txt else where in the current user directory.

**Comment by yingbull** [comment]
Is this no longer draft?

**Comment by pemba1s1** [comment]
@yingbull alex pointed out the read/write issue when storing in program files. So I'm looking into right now

**Comment by yingbull** [comment]
I am okay to go down the install just for me or all users option, but def for clinics with shared computers they won't want to have to install it in every docs account.

File: src/FreeScribe.client/utils/file_utils.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import ctypes
 import os
 import sys
 
 def get_file_path(*file_names: str) -> str:
```
[the remark this answers is not in the document]
Comment (pemba1s1) [reply src/FreeScribe.client/utils/file_utils.py:5]: done

File: src/FreeScribe.client/utils/file_utils.py [hunk 2]
```diff
@@ -1,17 +1,51 @@
...
```
[the remark this answers is not in the document]
Comment (pemba1s1) [reply src/FreeScribe.client/utils/file_utils.py:6]: handled for different platform

**Review by yingbull — APPROVED** [review]

**Pull request merged**